test/libsolidity/syntaxTests/controlFlow/storageReturn/unimplemented_library.sol

```diff
@@ -0,0 +1,3 @@
+library L {
+    function f() public returns(uint[] storage);
```

Comment:
Don't you have to give it a name at least?

Reply:
It would have warned even without a name (we also don't want to have unnamed return arguments without an ``return`` statement).

Reply:
I can add a second function with a named return argument to both tests, though.

Reply:
Just did that.